src/electionguard_cli/setup_election/output_setup_files_step.py

```diff
@@ -67,7 +67,7 @@ def _export_guardian_records(self, setup_inputs: SetupInputs, out_dir: str) -> N
         self.print_value("Guardian records", guardian_records_dir)
 
     def _export_guardian_private_keys(self, inputs: SetupInputs, keys_dir: str) -> None:
-        if path.exists(keys_dir):
+        if path.exists(keys_dir) and not inputs.force:
```

Comment:
I didn't end up using the force option, but I like having the option, so I left this in